dags/common/parsing/xml_extractors.py

```diff
@@ -28,7 +28,8 @@ def _get_text_value(self, raw_value):
         try:
             return raw_value.text
         except AttributeError:
-            return None
+            self.logger.error(f"{self.destination} is not found in XML")
```

Comment:
add a doi to the log msg if possible, cause if something breaks, you will search for the doi value  in logs probably

Reply:
I am not sure if can I do it. The values attribute values are set on __init__, so if I would pass the doi in extractors, it will be always None. Is my thinking makes sense?

Reply:
I see, maybe add it to the log msgs in iop parser then?

Reply:
I will. Until now, I didn't use the TextExtractor in IOP parsing, I will do in the upcoming PR for title and subtitle